### ISSUE TITLE
MOCKUP - Caltrans running sprint goals

### ISSUE THREAD

**Issue by jkarpen** [cagov/caldata-mdsa-caltrans-pems#254]
Sprint 2024-013 Goals:

Imputation
Research
Metrics
Optimization
Data Relay Server

- [ ] https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/170
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/186
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/195
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/115 (depends on https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/114 )
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/185
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/211
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/233
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/226

Stretch goals:

 

- [ ] https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/70 Start exploring how to load LCS / CHP data
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/71 lower priority after VDS metadata
- [ ]  Advanced Metrics:
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/187 waiting on https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/186
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/204
- [ ]  https://github.com/cagov/caldata-mdsa-caltrans-pems/issues/103
